docs_source_files/content/webdriver/support_classes.en.md

```diff
@@ -1,5 +1,76 @@
 ---
 title: "Support classes"
-weight: 5
+weight: 4
 ---
 
+Webdriver support classes are provided to make updating and maintaining your code easier. They provide a nice abstraction to make modelling HTML element(s) as domain objects easier, 
+also providing helper methods to make using such objects easy to reason about.
+We will learn about Locator Strategies, Events, PageFactory, LoadableComponent, ThreadGuard etc.
+
+Let's Start:
+
+## **ThreadGuard**
```

Comment:
This is only available in Java, we should mention that.

Reply:
Info panel added